Skyline.DataMiner.CICD.Tools.Validator/Skyline.DataMiner.CICD.Tools.Validator.csproj

```diff
@@ -39,8 +39,14 @@
     <PackageReference Include="Skyline.DataMiner.CICD.Tools.Reporter" Version="1.0.1" />
     <PackageReference Include="Skyline.DataMiner.XmlSchemas.Protocol" Version="1.0.2" />
     <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
+    <PackageReference Include="NuGet.Common" Version="6.6.1" ExcludeAssets="runtime" />
+    <PackageReference Include="NuGet.Configuration" Version="6.6.1" ExcludeAssets="runtime" />
+    <PackageReference Include="NuGet.Frameworks" Version="6.6.1" ExcludeAssets="runtime" />
+    <PackageReference Include="NuGet.Packaging" Version="6.6.1" ExcludeAssets="runtime" />
+    <PackageReference Include="NuGet.Protocol" Version="6.6.1" ExcludeAssets="runtime" />
+    <PackageReference Include="NuGet.Versioning" Version="6.6.1" ExcludeAssets="runtime" />
```

Comment:
Use 6.6.2 as 6.6.1 has vulnerabilities (I think it's in Frameworks?)

Reply:
updated